SDL 0244 - Custom Playback Rate (#108)
smartdevicelink/sdl_core_guides: docs/Integrating Your HMI/Developing the HMI UI/index.md

!!!

### Media Layout Elements

Apps which use the `MEDIA` template have access to a few specific UI elements that are not available to non-media apps. 

The following buttons can only be subscribed to by media apps and and are generally only available in the `MEDIA` template layout:

  - `PLAY_PAUSE`
  - `SEEKLEFT`
  - `SEEKRIGHT`
  - `TUNEUP`
  - `TUNEDOWN`

!!! NOTE

> Prior to RPC Spec version 5.0, the `OK` button name, which is available to all apps, was used for the purpose of play/pause toggling for media apps.

Reviewer: ```suggestion
Prior to RPC Spec version 5.0, the `OK` button name, which is available to all apps, was used for play/pause toggling in media apps.
```
